mythril/laser/ethereum/instructions.py

```diff
@@ -1837,6 +1833,20 @@ def stop_(self, global_state: GlobalState):
         """
         global_state.current_transaction.end(global_state)
 
+    @staticmethod
+    def _write_symbolic_memory(
```

Comment:
What do you think about `_write_symbolic_return_data`, write_symbolic_memory seems fairly generic